### ISSUE TITLE
ChatVertexAI has no property max_tokens.

### ISSUE THREAD

**Issue by daniel-deychakiwsky** [langchain-ai/langchain-google#596]
ChatVertexAI inherits from BaseChatModel which accepts a standard property, `max_tokens`, from the documentation. 

When initializing a ChatVertexAI instance with this standard property, it is not available on the instance and throws an error upon accessing. It is instead available under `max_output_tokens`. 

It should be made available on the instance as the other chat model integrations (ChatOpenAI, ChatAnthropic, ...) all have this property available on the instance.

```python
model = ChatVertexAI(
    model="gemini-1.5-pro-002",
    temperature=0.0,
    max_tokens=500,
    top_p=1.0,
)

print(model.max_output_tokens)  # 500
print(model.max_tokens).  # AttributeError: 'ChatVertexAI' object has no attribute 'max_tokens'
```

---
LangChain package versions:

langchain-core==0.3.15
langchain-google-vertexai==2.0.7

**Comment by lkuligin** [comment]
good point, would you be open to send a PR (please, create an alias for the pydantic field - in the same way we do it for model/model_name).

**Comment by trebedea** [comment]
Is there a reason for he newly added `max_tokens` property not to have a setter? 
There are cases when you want to change the `max_tokens` attribute in subsequent calls, for example when you are interested only in  a `"Yes/No"` answer without any longer explanations. This happens in Nemo Guardrails and VertexAI in one of the few Langchain providers that does support this: https://github.com/NVIDIA/NeMo-Guardrails/issues/993
